### PR TITLE
update og tags for /dance, /learn, and /hourofcode/overview

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 1]
```diff
@@ -1,6 +1,17 @@
 ---
 title: Hour of Code Teacher Resources
 theme: responsive
+social:
+  "twitter:card": photo
+  "twitter:site": "@codeorg"
+  "twitter:url": "https://<%=request.host%>/hourofcode/overview"
+  "twitter:title": "Hour of Code"
+  "twitter:description": "The Hour of Code is a global movement reaching tens of millions of students. One-hour tutorials are available in 45+ languages for all ages."
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml]: Do we not want these loc'ed?

**Comment by breville** [comment]
The dance-social.png image provided has transparent pixels that were probably supposed to be cropped.  (I think it's the same one that's already going to be in /shared/ for the teacher announcement?

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml]: Thanks for the reminder! Let me update...

File: pegasus/src/social_metadata.rb [hunk 2]
```diff
@@ -19,7 +20,8 @@ def get_social_metadata_for_page(request)
     hoc_2018_creativity: {path: "/images/social-media/hourofcode-2018-creativity.jpg", width: 846, height: 529},
     hoc_student_challenge: {path: "/images/fit-1920/social-media/hoc-student-challenge.jpg", width: 1920, height: 1080},
     mc_social_2017: {path: "/images/mc/mc_social_2017.jpg", width: 1200, height: 630},
-    mc_social_2018: {path: "/images/social-media/mc-social-2018.jpg", width: 1200, height: 630}
+    mc_social_2018: {path: "/images/social-media/mc-social-2018.jpg", width: 1200, height: 630},
+    dance_2018: {path: "/images/social-media/dance-social.jpg", width: 1200, height: 630}
```
Comment (breville) [inline pegasus/src/social_metadata.rb]: The image provided is far smaller than these dimensions.

**Comment by tanyaparker** [comment]
Cropped image and updated dimensions. It's slightly different than the teacher announcement one because it has the CODE logo.

**Pull request merged**